examples/plot_darcy_flow_spectrum.py

```diff
@@ -0,0 +1,104 @@
+"""
+A simple Darcy-Flow spectrum analysis
+===========================
+In this example, we demonstrate how to use the spectrum analysis function on the small Darcy-Flow example.
+"""
+# Original Author: Zongyi Li
+# Modified by: Robert Joseph George
+# %%
+# Import the library
+# ------------------
+# We first import our `neuralop` library and required dependencies.
+import numpy as np
+import torch
+import matplotlib
+import matplotlib.pyplot as plt
+from neuralop.utils import spectrum2
+
+font = {'size'   : 28}
+matplotlib.rc('font', **font)
+
+from timeit import default_timer
+
+torch.manual_seed(0)
+np.random.seed(0)
+
+# %%
+# Define some variables
+T = 500 # number of time steps
+samples = 50
+s = 16 # resolution of the dataset
+S = s
+
+# additional paramaters for the dataset
+Re = 5000
+index = 1
+T = 100
+dataset_name = "Darcy Flow"
+
+# %%
+HOME_PATH = '/home/user/'
```

Comment:
@Robertboy18 this will not work for users -  better to just use the builtin dataset for this, as in https://github.com/neuraloperator/neuraloperator/blob/main/examples/plot_FNO_darcy.py

Reply:
That makes sense, will load the dataset directly then!

Reply:
Actually, the issue is that we need to load the dataset in its raw format (so .pt, .mat, .zarr etc) to get the full spectrum; we cannot use the data loaders! I can leave a comment asking the user to change the HOME_PATH as 'neuraloperator/neuralop/datasets/data/darcy_test_16.pt' this path is the direct link to the raw format dataset.

Reply:
Why?

Reply:
Changed it!